Source/WebCore/rendering/BreakLines.h

```diff
@@ -59,6 +59,12 @@ static const UChar lineBreakTableFirstCharacter = '!';
 static const UChar lineBreakTableLastCharacter = 127;
 static const unsigned lineBreakTableColumnCount = (lineBreakTableLastCharacter - lineBreakTableFirstCharacter) / 8 + 1;
 WEBCORE_EXPORT extern const unsigned char lineBreakTable[][lineBreakTableColumnCount];
+inline bool lineBreakTableUnsafeLookup(UChar before, UChar after) // range check before calling
```

Comment:
what makes it `unsafe`?

Reply:
It doesn't do any range checking before indexing into the table.